build.sbt

```diff
@@ -58,9 +58,8 @@ val compile: Seq[ModuleID] = Seq(
   "uk.gov.hmrc"       %% "bootstrap-frontend-play-26" % "5.4.0",
   "uk.gov.hmrc"       %% "play-frontend-govuk"        % "0.77.0-play-26",
   "uk.gov.hmrc"       %% "play-frontend-hmrc"         % "0.76.0-play-26",
```

Comment:
Not related to your changes, but could you please still remove GovukWrapper from the list of packages excluded from test coverage as this doesn't exist anymore?